### PR TITLE
Added html for section 2

### PR DESCRIPTION
**Pull request by Alpha787**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -31,23 +31,24 @@ <h1 class="download__title">
       </div>
     </div>
   </section>
-  <section class="section" id="warranty">
-    <div class="wrapper">
-      <div class="text-block">
-        <h1 class="title">Peaceful shopping mindful money</h1>
-        <p class="copy">
-          See how we can help with making your shopping experience and money management more ZEN.
+  <!-- ������ 2 -->
```
Comment (Palych18) [inline index.html:34]: Сorrect the comment

**Review by Palych18 — APPROVED** [review]
Done

File: index.html (hunk 1 above)
Comment (nagorapro) [inline index.html:34]: Delete or replace

````
<!-- Section 2 -->

File: index.html
Comment (nagorapro) [inline index.html:1]: Don't use the formatter yet

**Review by nagorapro — COMMENTED** [review]
Need fix

**Pull request merged**